### PR TITLE
Where is the animal? dynamic variant

### PR DESCRIPTION
**Pull request by verafili**
A new game variant of _Where is the animal?_ game.

Also, there are new graphical statistics for games with dynamic adaptation. Instead of the area chart, information about levels variations during the gameplay is plotted for dynamic variants of Ninja and Memory games. For Where is the animal, a table allows determining if a child's decisions were random or not after every eight rounds depending on the alpha (probability of error).

### CODE REVIEW

File: gazeplay-core/src/main/resources/data/common/table.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.table-view .column-header {
+    -fx-text-fill: white;
+}
+
+.table-view .column-header-background .label {
+    -fx-background-color: transparent;
+    -fx-text-fill: white;
+}
```
Comment (Noars) [inline gazeplay-core/src/main/resources/data/common/table.css:17]: Need new line at the end of the file

**Review by PillouMatou — COMMENTED** [review]
it work well, but why in the game where is the animal we only look for cats?

**Review by Noars — COMMENTED** [review]
The game work fine but is it normal that we are always asked to find the cat and not another animal?

**Comment by kuhlkrein** [comment]
When the "winner" image is being displayed on fullscreen, it is stretching a lot on the horizontal axis but not on the vertical, it looks like this issue has been fixed for 2*2+ variant but not for the 2*2 (even in normal mode), we need to fix this (maybe in a next PR as it is probably not an error due to your code)

**Review by kuhlkrein — CHANGES_REQUESTED** [review]
Please try to rebase to not display the "fixation length" commit in this PR

**Review by kuhlkrein — APPROVED** [review]
looks good to me

**Pull request merged**